public/robots.txt

```diff
@@ -0,0 +1 @@
+Sitemap: https://ifsc.stage.razorpay.in/sitemap.xml.gz
```

Comment:
put the prod url. until we submit the sitemap, search engines will not crawl i guess

Reply:
Cool.